source/screens/caseScreens/CaseLogic.js

```diff
@@ -26,17 +26,30 @@ export const Status = {
 };
 const oldCaseLimit = 4 * 30 * 24 * 60 * 60 * 1000; // cases older than 4 months are classified as old.
 
+const getIds = async (formTypes, findFormByType) =>
+  formTypes.map(async type => {
+    const formSummary = await findFormByType(type);
+    return formSummary.id;
+  });
+
 /**
  * Takes a typeCase and an array of cases and returns [status, latestCase, relevantCases].
  * @param {obj} caseType an object with {name, forms: [formIds], icon: icon name, navigateTo: string with the navigation path}.
  * @param {[cases]} cases array of case objects.
  */
-export const getCaseTypeAndLatestCase = (caseType, cases) => {
+export const getCaseTypeAndLatestCase = async (caseType, cases, findFormByType) => {
```

Comment:
If i understand this function correct, we do not return any caseType? So might be a good idea to rename it to what we actually are getting back. Or is caseType what we refer to as the status of a case here?

It's confusing to read either way.